demo/summer-2016.html

```diff
@@ -244,11 +244,26 @@
         langTools.addCompleter(texCompleter);
         console.log("Simple Latex Completer successfully added");
     });
-    
-    document.getElementById("editor").addEventListener("click", function() {
-        highlighter.highlightBrackets(editor);
+
+    var keyboardHandler = {
+      name: 'highlight',
+      bindKey: {win: 'Ctrl-0', mac: 'Commad-0'},
+      exec: function(editor) { return highlighter.highlightBrackets(editor)},
+      readOnly: true
+    }
+    editor.commands.addCommand(keyboardHandler);
+
+    document.getElementById("editor").addEventListener("click", function(editor){
```

Comment:
Yes, it is need to screen is cleared when the cursor is moved to another location.